u-root.go

```diff
@@ -58,6 +59,7 @@ func init() {
 	noCommands = flag.Bool("nocmd", false, "Build no Go commands; initramfs only")
 
 	flag.Var(&extraFiles, "files", "Additional files, directories, and binaries (with their ldd dependencies) to add to archive. Can be speficified multiple times.")
+	tags = flag.String("tags", "", "a space-separated list of build tags to consider satisfied during the build.")
```

Comment:
Maybe call it build-tags? Just so it's clear

Reply:
done